### PR TITLE
Validating the dashboard name paramater

### PR DESCRIPTION
**Pull request by andreaswittig**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by andreaswittig** [comment]
test

File: test/test.js [hunk 1]
```diff
@@ -11,3 +11,16 @@ test('defaults', async t => {
     t.pass();
   }
 });
+
+test('invalid dashboard name', async t => {
+  const stackName = cfntest.stackName();
+  try {
+    t.log(await cfntest.createStack(`${__dirname}/invalid-dashboard-name.yml`, stackName, {}));
+    // what could we test here?
+    t.log(await cfntest.deleteStack(stackName));
```
Comment (michaelwittig) [inline test/test.js]: please move the deleteStack into finally

**Review by michaelwittig — CHANGES_REQUESTED** [review]

File: test/test.js (hunk 1 above)
Comment (andreaswittig) [reply test/test.js]: 👍

**Comment by andreaswittig** [comment]
test

**Review by michaelwittig — APPROVED** [review]

**Comment by andreaswittig** [comment]
test

**Pull request merged**